morph.go

```diff
@@ -464,7 +479,13 @@ func buildHosts(hosts []nix.Host) (resultPath string, err error) {
 		return
 	}
 
-	resultPath, err = nix.BuildMachines(evalMachinesPath, deploymentPath, hosts, nixBuildArg)
+	nixBuildTargets := ""
+	if path, err := filepath.Abs(nixBuildTargetFile); err == nil {
```

Comment:
@Shados Should probably be:
```
	if nixBuildTargetFile != "" {
		if path, err := filepath.Abs(nixBuildTargetFile); err == nil {
			nixBuildTargets = fmt.Sprintf("import \"%s\"", path)
		}
	} else if nixBuildTarget != "" {
		nixBuildTargets = fmt.Sprintf("{ \"out\" = %s; }", nixBuildTarget)
	}
```

(Or some prettier solution with some more variables), but at least the situation where `nixBuildTargetFile` is unset must be guarded. :)